### ISSUE TITLE
number of GPUs used during the training

### ISSUE THREAD

**Issue by ttgeng233** [bytedance/video-SALMONN-2#11]
For the training of video-salmonn2 plus, may I ask how many GPUs you used in each training stage? 
This information is important to ensure that the total batch size remains unchanged, thereby avoiding any impact on the training results. (总total batch size = batch size per gpu × num_GPU × gradient_accumulation_steps)

**Comment by DragonAura** [comment]
GPU number and batch size per GPU used in each stage remain the same as the original video-SALMONN 2, which is detailed in Table 1 in the paper. We do not use gradient accumulation in all stages. Additionally, when using video-SALMONN 2+ 7B generated data to finetune video-SALMONN 2+ 72B, we use 64 GPUs and batch size on each device = 1 for larger VRAM to train the 72B model.
